### ISSUE TITLE
Spacing around ranges edge case with =, results in invalid code

### ISSUE THREAD

**Issue by jessesquires** [nicklockwood/SwiftFormat#180]
#### Using: 

`--ranges nospace`

#### Results:

```swift
let range =..<String.Index(encodedOffset: limit)
```

#### Compiler error:

```
Expected expression after unary operator
'=' must have consistent whitespace on both sides
```

#### Expected:

```swift
let range = ..<String.Index(encodedOffset: limit)
```

**Comment by nicklockwood** [comment]
Ah, thanks - I’ll get it sorted asap.

**Comment by jessesquires** [comment]
Thanks @nicklockwood ! 😄

**Comment by nicklockwood** [comment]
@jessesquires fixed in 0.29.8
